StripePayments/StripePayments/Source/API Bindings/STPAPIClient+Payments.swift

```diff
@@ -941,6 +941,103 @@ extension STPAPIClient {
     }
 }
 
+extension STPAPIClient {
+    @_spi(STP) public func listPaymentMethods(
+        forCustomer customerID: String,
+        using ephemeralKeySecret: String,
+        types: [STPPaymentMethodType] = [.card],
+        completion: @escaping STPPaymentMethodsCompletionBlock
+    ) {
+        let header = authorizationHeader(using: ephemeralKeySecret)
+        // Unfortunately, this API only supports fetching saved pms for one type at a time
```

Comment:
I think this comment was copy and paste, but I believe that type is used as an optional filter:
https://stripe.com/docs/api/payment_methods/customer_list

`type` seems to be an optional parameter - Seems like at the very least, we should update this comment with a TODO: comment to update our code to fetch all payment methods, and then filter payment methods locally to avoid multiple round trips.

Reply:
Ooh, yeah, the comment made me think this was non-optional. We should probably fetch all PMs and filter them on the client side in PaymentSheet too.